app/javascript/src/scenes/bill/components/Content.js

```diff
@@ -1,15 +1,15 @@
 // @flow
 import React, { Component } from 'react'
 import { createFragmentContainer, graphql } from 'react-relay'
-import moment from 'moment'
+import { default as parseDate } from 'date-fns/parse'
+import differenceInHours from 'date-fns/difference_in_hours'
+import { default as dateFormat } from 'date-fns/format'
```

Comment:
Unless you're opposed, can we call this `formatDate`? I try to name all methods starting with a verb.

Reply:
👍